src/javascripts/components/auth/auth.js

```diff
@@ -7,15 +7,39 @@ const signMeIn = () => {
 };
 
 const loginButton = () => {
+  $('#app').html('');
   const domString = `
-  <div id="auth">
-    <span style="font-size: 9em; color: Red;"><i class="fab fa-empire"></i>
-    <h4>Empire Dasboard</h4>
-    <button id="google-auth" class="btn btn-danger"><i class="fab fa-google"></i></i>oogle Login
-    </button>
-  </div>`;
+  <nav class="navbar fixed top navbar-expand-lg navbar-light bg-light">
+    <a class="navbar-brand" href="#">Star Destroyer Dashboard</a>
+      
+  <button id="google-auth" class="btn btn-warning btn-lg">
+        <i class="fab fa-google"></i></i>oogle Login
+        </button>
+  </nav>
 
-  $('#auth').html(domString);
+  <div class="logoutCards">
```

Comment:
Why are these cards hard coded? These should be pulling directly from your API.

Reply:
This has been addressed.